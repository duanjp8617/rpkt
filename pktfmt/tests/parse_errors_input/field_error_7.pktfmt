packet FakeProto {
    header = [
        field1 = Field {
            bit = 32,
            repr = &[u8],
            default = @[4,4,4,4]
        },
        field2 = Field {
            bit = 1,
            arg = bool,
            default = @true
        },
        field3 = Field {
            bit = 7
        }
    ]
}

