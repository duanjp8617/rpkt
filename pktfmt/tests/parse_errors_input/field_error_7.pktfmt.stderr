ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/field_error_7.pktfmt 3:18-7:9
                     ~~~~~~~
3 |         field1 = Field {
4 |             bit = 32,
5 |             repr = &[u8],
6 |             default = @[4,4,4,4]
7 |         },
    ^^^^^^^^^
field error 7:
default can not be fixed for bool value and byte array
